README.rst

```diff
@@ -261,56 +261,61 @@ Let us check the connection:
 .. code-block:: console
 
    $ reana-client ping
-   Server is running.
+   Connected to http://192.168.99.100:31201 - Server is running.
```

Comment:
`Connected to http://reana.cern.ch - Server is running.`

Reply:
Same as https://github.com/reanahub/reana-demo-worldpopulation/pull/21#discussion_r208120932 regarding access token.